script/install-local-registry.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+CONTROL_PLANE_CONTAINER="kubeapps-ci-control-plane"
+REGISTRY_NS=ci
+DOCKER_REGISTRY_HOST=local-docker-registry
+DOCKER_REGISTRY_PORT=5600
+DOCKER_REGISTRY_URL="https://$DOCKER_REGISTRY_HOST:$DOCKER_REGISTRY_PORT"
+DOCKER_USERNAME=testuser
+DOCKER_PASSWORD=testpassword
+
+installLocalRegistry() {
+    PROJECT_PATH=$1
+
+    kubectl create namespace $REGISTRY_NS
+
+    # Add our CA to the node
+    docker cp "$(mkcert -CAROOT)"/rootCA.pem "$CONTROL_PLANE_CONTAINER:/usr/share/ca-certificates/local-rootCA.pem"
+    docker exec --user root $CONTROL_PLANE_CONTAINER sh -c "echo 'local-rootCA.pem' >> /etc/ca-certificates.conf"
+    docker exec --user root $CONTROL_PLANE_CONTAINER update-ca-certificates -f
+
+    # Restart containerd to make the CA addition effective
+    docker exec --user root $CONTROL_PLANE_CONTAINER systemctl restart containerd
+
+    # Generate new certificate for registry
+    mkcert -key-file $PROJECT_PATH/devel/localhost-key.pem -cert-file $PROJECT_PATH/devel/docker-registry-cert.pem $DOCKER_REGISTRY_HOST "docker-registry.$REGISTRY_NS.svc.cluster.local"
+    kubectl -n $REGISTRY_NS delete secret registry-tls --ignore-not-found=true
+    kubectl -n $REGISTRY_NS create secret tls registry-tls --key $PROJECT_PATH/devel/localhost-key.pem --cert $PROJECT_PATH/devel/docker-registry-cert.pem
```

Comment:
As above, I wonder if it's worth creating the `registry-auth` here as well so you don't need to repeat the encoded secret in the yaml.